src/Sarif/Autogenerated/Result.cs

```diff
@@ -50,6 +50,12 @@ public SarifNodeKind SarifNodeKind
         [DataMember(Name = "message", IsRequired = false, EmitDefaultValue = false)]
         public Message Message { get; set; }
 
+        /// <summary>
+        /// A string that identifies the message within the metadata for the rule used in this result.
+        /// </summary>
+        [DataMember(Name = "ruleMessageId", IsRequired = false, EmitDefaultValue = false)]
+        public string RuleMessageId { get; set; }
```

Comment:
>RuleMessageId [](start = 22, length = 13)

@michaelcfanning ?? It looks like you _added_ `RuleMessageId`, but we just got finished removing it.

Reply:
Oh, I'm not looking at the latest commit.

---
In reply to: [215004356](https://github.com/Microsoft/sarif-sdk/pull/1067#discussion_r215004356) [](ancestors = 215004356)